crypto/fipsmodule/ec/ec_key.c

```diff
@@ -421,7 +421,7 @@ int EC_KEY_check_fips(const EC_KEY *key) {
 end:
   FIPS_service_indicator_unlock_state();
   if(ret){
-    FIPS_service_indicator_update_state();
+    EC_KEY_keygen_verify_service_indicator((EC_KEY*)key);
```

Comment:
Just curious: why was the cast needed here and not in the other change?

Reply:
the function argument in this function is declared as `const` so the compiler complains that we are dropping the const qualifier.